clang-tools-extra/clang-tidy/readability/ContainerContainsCheck.cpp

```diff
@@ -32,7 +33,8 @@ void ContainerContainsCheck::registerMatchers(MatchFinder *Finder) {
 
   const auto FindCall =
       cxxMemberCallExpr(
-          argumentCountIs(1),
+          anyOf(argumentCountIs(1),
+                allOf(argumentCountIs(2), hasArgument(1, cxxDefaultArgExpr()))),
```

Comment:
Maybe a comment here saying `std::string` and `std::string_view` take an optional `pos` argument on where to start the search. Also match `ignoringParenImpCasts(integerLiteral(0))`?

Reply:
I tried to match the two arguments but I fail to remove the second argument in the Fixit hint. Could you suggest how to get the location after the first argument. I tried binding the first argument (e.g. "test"), but the source range seems to be the first character (") instead of the whole argument ("test").

Reply:
Considering `str.find('a') != std::string::npos`. Instead of deleting stuff from `)` to `npos`, you could replace everything from after `'a'` to the end of the comparison with `)`, that should work. I wrote something like this here:
https://github.com/llvm/llvm-project/blob/e8f01b0557354a28d17bfe618df5e257ec3e982a/clang-tools-extra/clang-tidy/modernize/UseStartsEndsWithCheck.cpp#L159-L165